metagraph-plugin/tests/test_types.py

```diff
@@ -32,40 +32,34 @@ def test_edge_schema(katanagraph_rmat15_cleaned_di):
 
 def test_edge_property(katanagraph_rmat15_cleaned_di):
     assert katanagraph_rmat15_cleaned_di.value.loaded_edge_schema()[0].name == "value"
-    assert katanagraph_rmat15_cleaned_di.value.get_edge_property(0) == katanagraph_rmat15_cleaned_di.value.get_edge_property("value")
+    assert katanagraph_rmat15_cleaned_di.value.get_edge_property(
+        0
+    ) == katanagraph_rmat15_cleaned_di.value.get_edge_property("value")
     assert katanagraph_rmat15_cleaned_di.value.get_edge_property("value").to_pandas()[0] == 339302416426
 
 
 def test_topology(katanagraph_rmat15_cleaned_di):
-    assert katanagraph_rmat15_cleaned_di.value.edges(0) == range(0, 20767)
-    assert [katanagraph_rmat15_cleaned_di.value.get_edge_dest(i) for i in katanagraph_rmat15_cleaned_di.value.edges(0)][0:5] == [
-        1,
-        2,
-        3,
-        4,
-        5,
-    ]
-    assert katanagraph_rmat15_cleaned_di.value.edges(8) == range(36475, 41133)
-    assert [katanagraph_rmat15_cleaned_di.value.get_edge_dest(i) for i in katanagraph_rmat15_cleaned_di.value.edges(8)][0:5] == [
-        0,
-        9,
-        10,
-        11,
-        12,
-    ]
+    assert katanagraph_rmat15_cleaned_di.value.edge_ids(0) == range(0, 20767)
+    assert [
+        katanagraph_rmat15_cleaned_di.value.get_edge_dest(i) for i in katanagraph_rmat15_cleaned_di.value.edge_ids(0)
+    ][0:5] == [1, 2, 3, 4, 5,]
+    assert katanagraph_rmat15_cleaned_di.value.edge_ids(8) == range(36475, 41133)
+    assert [
+        katanagraph_rmat15_cleaned_di.value.get_edge_dest(i) for i in katanagraph_rmat15_cleaned_di.value.edge_ids(8)
+    ][0:5] == [0, 9, 10, 11, 12,]
 
 
-def test_num_nodes(networkx_weighted_undirected_8_12, networkx_weighted_directed_8_12):
+def test_num_nodes_networkx(networkx_weighted_undirected_8_12, networkx_weighted_directed_8_12):
     assert len(list(networkx_weighted_undirected_8_12.value.nodes(data=True))) == 8
     assert len(list(networkx_weighted_directed_8_12.value.nodes(data=True))) == 8
 
 
-def test_num_nodes(networkx_weighted_undirected_8_12, networkx_weighted_directed_8_12):
+def test_num_nodes_networkx_2(networkx_weighted_undirected_8_12, networkx_weighted_directed_8_12):
```

Comment:
test_num_edges_networkx?

Reply:
Ooops. Thanks. I couldn't figure out the difference between the tests and apparently I was just momentarily blind.